galaxy/wrapper/computeMatrix.xml

```diff
@@ -85,12 +79,14 @@
                 help="Label to use in the output."/>
         </repeat>
 
-        <param name="scoreFile" format="bigwig" type="data"
```

Comment:
Alternative. Please note the multiple=True.
Moreover, why does the `-scoreFile` only have one dash? Needs a fix.
```
<param argument="-scoreFile" format="bigwig" type="data" multiple="True"
           label="Score file"
           help="Should be a bigWig file (containing a score, usually covering
           the whole genome). You can generate a bigWig file either from a
           bedGraph or WIG file using UCSC tools or from a BAM file using the
           deepTool bamCoverage."/>
```